### DISCUSSION TITLE
When I upgraded the database to Banyan0.7.0,Skywalking threw an IO exception.

### DISCUSSION THREAD

**Discussion by xiangzirui** [apache/skywalking#12828] [Q&A]
I deployed 10.1.0 sykwalking and 0.7.0 storage server BanyanDB through helm this week.
At the beginning of the deployment, the system runs normally.
After using it for a period of time, the system began to report IO exception frequently, and the data query could not get the results,
and then the UI interface could not be accessed, and the whole line collapsed.
After that, I redeployed it twice, and I encountered the same problem on other machines. What bothers me is that I can't find out the problem in the contents of the log. Please help me recover. Thank you.

![3f837b5412be481c46e0f85f22504b32](https://github.com/user-attachments/assets/a35ce9ab-9a13-4870-86c1-6eabc09c93b3)
![image](https://github.com/user-attachments/assets/3843fcf3-982e-4675-b3ab-a81d916e38ba)

By the way, I would also like to ask what I should do if I want to specify the monitoring time while jumping to the service page. 
Thank you for your answer.
@wu-sheng

**Comment by wu-sheng** [comment]
BanyanDB 0.7.1 is out for some fixes. You could try that.

**Reply by xiangzirui** [reply]
> BanyanDB 0.7.1 已发布，包含一些修复。您可以尝试一下。

I deployed with helm4.7.0. If I want to upgrade the database version, what should I do? Do I directly update the mirror version of the database in k8s, or do I need to modify the helm chart file?

**Reply by wu-sheng** [reply]
From 0.7 to 0.7.1, you just need to update the image tag. Nothing more.
But you are better to remove all existing data files. We don't verify compatibility for now.
